### ISSUE TITLE
Include configuration files in Cluster Support Bundle

### ISSUE THREAD

**Issue by kimoswalt** [Graylog2/graylog2-server#20190]
## What?
It would be beneficial if the Cluster Support Bundle included the server.conf/gralog.conf file. 
It would be a bonus if it also  included the Graylog JVM settings:
/etc/default/graylog-server/jvm.options

## Why?
Including the config files would save time for both the support team and the customer when troubleshooting. 

## Your Environment
 Would be beneficial for any environment

**Comment by rkmbaxed** [comment]
Aren't there a lot of users and passwords in the configuration that shouldn't be shared? (mongodb, opensearch, certificates,...)

**Comment by bernd** [comment]
@rkmbaxed @kimoswalt Correct! We can't include the `server.conf`/`graylog.conf` file in the support bundle. Since the bundle is generated inside the server, we can include non-sensitive settings without reading the actual files from the disk. (which might also contain other sensitive information in comments)
